examples/example.js

```diff
@@ -15,7 +15,8 @@ const schema = {
 
 fastify
   .get('/', schema, function (req, reply) {
-    reply(null, { hello: 'world' })
+    reply.header('Content-Type', 'application-json').code(200)
+    reply.reply(null, { hello: 'world' })
```

Comment:
Having the error suggest that it is a callback. But it is not, as it does not hold the state of the response.

Let's do it the Hapi way, and avoid positional arguments. If the first arg is not the error, no problem. I think it would simplify things.